src/main/java/zd/zero/waifu/motivator/plugin/promotion/MemePromotionDialog.kt

```diff
@@ -93,6 +96,7 @@ class AniMemePromotionDialog(
 
             override fun actionPerformed(e: ActionEvent) {
                 installAndEnable(
+                    project,
```

Comment:
Got a compatibility problem

```
Plugin zd.zero.waifu-motivator-plugin:2.3.0 against IC-213.5744.223: 1 compatibility problem. 1 usage of internal API
Compatibility problems (1): 
    #Invocation of unresolved method com.intellij.openapi.updateSettings.impl.pluginsAdvertisement.PluginsAdvertiser.installAndEnable$default(Project, Set, boolean, boolean, ModalityState, Runnable, int, Object) : void
        Method zd.zero.waifu.motivator.plugin.promotion.AniMemePromotionDialog.buildInstallAction$1.actionPerformed(java.awt.event.ActionEvent e) : void contains an *invokestatic* instruction referencing an unresolved method com.intellij.openapi.updateSettings.impl.pluginsAdvertisement.PluginsAdvertiser.installAndEnable$default(com.intellij.openapi.project.Project, java.util.Set, boolean, boolean, com.intellij.openapi.application.ModalityState, java.lang.Runnable, int, java.lang.Object) : void. This can lead to **NoSuchMethodError** exception at runtime.
I
```

How about we bump it up to the minimum version this shows up in?

Reply:
Sure, having some trouble figuring out which version is required for this. Couldn't find anything in the [source code](https://github.com/JetBrains/intellij-community/blob/idea/222.3739.54/platform/platform-impl/src/com/intellij/openapi/updateSettings/impl/pluginsAdvertisement/PluginsAdvertiser.kt). Have asked for help in the slack channel

Reply:
Looks like maybe seven months ago?

<img width="1016" alt="Screen Shot 2022-10-04 at 5 25 34 AM" src="https://user-images.githubusercontent.com/15972415/193796426-aeb641a9-00fd-405b-86c5-4ed0ee933ac7.png">

My tip would just be to start from the lowest version & just keep updating the version increments until it stops complaining. You can run the verifier locally by running gradle `runPluginVerifier` task.

Reply:
had to increment min version to 2022.1

Reply:
That's fair, it would have had to have happened some time. Now's a good a time as any.